packages/mobile/src/components/fields/PriceField.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState, useEffect, useCallback } from 'react'
```

Comment:
headsup, there is a TrackPriceField.tsx for the upload flow, wondering if these can be combined in some way? the TrackPriceField needs some work, so using what you have here would be clutch. it looks like BoxedTextField might not implement super cleanly, so lmk

Reply:
That's actually where I got the implementation for this! The longer term plan was to make both of them share a component. But the PR was already getting a little large. I can follow up and switch it over. There is a similar component split on web from a previous PR.